### PR TITLE
Emissives and plane master refactor.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
~~Hamfisted half-understood implementation based on /tg/'s version. Currently not working at all.~~ Abandoned attempt at porting /tg/ emissives, now just cleaning up some of the existing systems.

### CODE REVIEW

**Review by comma — DISMISSED** [review]

**Review by comma — DISMISSED** [review]

**Comment by Lohikar** [comment]
Going to block this for a day or so -- intend to fix the O7L conflicts today or tomorrow, and this will conflict pretty hard with that.

**Comment by Lohikar** [comment]
(If I've got nothing on Wednesday [2 days], feel free to remove the DNM)

File: code/modules/detectivework/tools/uvlight.dm [hunk 1]
```diff
@@ -39,11 +39,10 @@
 /obj/item/uv_light/on_update_icon()
 	cut_overlays()
 	if(on)
-		var/image/I = image(icon, "[icon_state]-on")
-		if(plane != HUD_PLANE)
-			I.layer = ABOVE_LIGHTING_LAYER
-			I.plane = EFFECTS_ABOVE_LIGHTING_PLANE
-		add_overlay(I)
+		if(plane == HUD_PLANE)
+			add_overlay(image(icon, "[icon_state]-on"))
```
Comment (Lohikar) [inline code/modules/detectivework/tools/uvlight.dm]: `add_overlay("[icon_state]-on")`, no?

File: code/modules/mob/living/simple_animal/constructs/constructs.dm [hunk 2]
```diff
@@ -61,9 +62,9 @@
 	qdel(src)
 
 /mob/living/simple_animal/construct/on_update_icon()
-	overlays.Cut()
 	..()
-	add_glow()
+	cut_overlays()
+	set_overlays(emissive_overlay(icon,"glow-[icon_state]"))
```
Comment (Lohikar) [inline code/modules/mob/living/simple_animal/constructs/constructs.dm:66]: Don't need `cut_overlays()` if using `set_overlays()`.

**Review by Lohikar — APPROVED** [review]

**Pull request merged**